### PR TITLE
Home page partials

### PR DESCRIPTION
**Pull request by abacz**
*** PR for UI System purposes ***

### CODE REVIEW

File: app/javascript/stylesheets/bootstrap-grid.min.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
...
+/*# sourceMappingURL=bootstrap-grid.min.css.map */
```
Comment (goreck888) [inline app/javascript/stylesheets/bootstrap-grid.min.css]: Are you sure, minified files of bootstrap grid should be included in `app` directory?

Comment (abacz) [reply app/javascript/stylesheets/bootstrap-grid.min.css]: Thanks for your comment. This PR lives and we are checking different solutions.

**Comment by agpul** [comment]
1. The filter section is bolded:
image

**Comment by agpul** [comment]
2. The search bar is short: 
image

**Comment by agpul** [comment]
3. My EOSC Marktplace is different here and an additional Logout option is available(the difference between production instance) 
image

**Comment by agpul** [comment]
4. Pagination: 
image

**Comment by agpul** [comment]
5. I am logged out, I am clicking on MY EOSC Marketplace and nothing happens 
image

**Comment by agpul** [comment]
6. 
image

**Comment by agpul** [comment]
7. Offers boxes have different sizes
image

**Comment by agpul** [comment]
8. This upper navigation bar is broken (configuration should be grey, moreover when I click on "Configuration" I don't have an arrow but rectangle)
image

**Comment by agpul** [comment]
9. Parameters on Configuration step look bad:
image

**Comment by agpul** [comment]
10. This should not be bolded:
image

**Comment by agpul** [comment]
11. Project form - headings should not be bolded:
image

**Comment by agpul** [comment]
12. Service description looks bad:
image

vs production MP:
image

**Comment by agpul** [comment]
13. image

**Comment by agpul** [comment]
13 is OK now
12. When I have bullet points the description of the service looks ok, but if I have numbers point:
Production:
image
vs PR:
image


14. Please have a look. When I type something in search bar in Backoffice, All services section is moving down:
image

**Comment by agpul** [comment]
I enclose the image in one category in backoffice and look what happened:
image

**Comment by abacz** [comment]
> I enclosed the image in one category in backoffice and look what happened:
> image

Fixed

**Comment by agpul** [comment]
I found the additional bug:
image
The same situation occurs on use-cases and learn_more sections.
I am sure that it was working properly before so we need to investigate...

**Comment by abacz** [comment]
> I found the additional bug:
> The same situation occurs on use-cases and learn_more sections.
> I am sure that it was working properly before so we need to investigate...

Static height set.

File: .haml-lint.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 linters:
   ViewLength:
     enabled: true
-    max: 500
+    max: 620
```
Comment (mkasztelnik) [inline .haml-lint.yml]: Heh - but at what cost :)

**Pull request merged**